templates/admin/auth/edit.tmpl

```diff
@@ -434,6 +434,17 @@
 				</div>
 			</form>
 		</div>
+
+		<h4 class="ui top attached header">
+			{{.locale.Tr "admin.auths.tips"}}
+		</h4>
+		<div class="ui attached segment">
+			<h5>GMail Settings:</h5>
+			<p>Host: smtp.gmail.com, Port: 587, Enable TLS Encryption: true</p>
+
+			<h5>{{.locale.Tr "admin.auths.tips.oauth2.general"}}:</h5>
+			<p>{{.locale.Tr "admin.auths.tips.oauth2.general.tip"}}</p>
+		</div>
```

Comment:
Screenshot of this?

Reply:
It's just copied from the "new" page. Nothing special.

![image](https://github.com/go-gitea/gitea/assets/2114189/1bfe84df-8783-4bec-913a-5c2c9410e365)